packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -146,6 +155,15 @@ describe('KeyringController', () => {
           );
         });
       });
+
+      it('should emit accountAdded', async () => {
+        await withController(async ({ controller, messenger }) => {
+          const listener = sinon.spy();
+          messenger.subscribe('KeyringController:accountAdded', listener);
+          const { addedAccountAddress } = await controller.addNewAccount();
+          expect(listener.calledWith(addedAccountAddress)).toBe(true);
```

Comment:
Perhaps dividing these tests into Arrange, act, and assert sections would make them more readable and highlight what we are testing in each case

More info on our [contributor docs](https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#-highlight-the-exercise-phase)